comfy_cli/command/run/__init__.py

```
@@ -0,0 +1,824 @@
+    record. The record's ``outputs`` map is keyed by node id."""
+    outputs = record.get("outputs") or {}
+    if not isinstance(outputs, dict):
+        return 0
+    return sum(1 for v in outputs.values() if isinstance(v, dict) and v)
+
+
+def execute_cloud(
+    workflow: str,
+    *,
+    wait: bool = False,
+    verbose: bool = False,
```

Pylint (4.0.5)

[warning] Unused argument 'verbose'

(W0613)